buku.py

```diff
@@ -2054,6 +2054,14 @@ def auto_import_from_browser(self):
             GC_BM_DB_PATH = '~/.config/google-chrome/Default/Bookmarks'
             CB_BM_DB_PATH = '~/.config/chromium/Default/Bookmarks'
 
+            DEFAULT_FF_FOLDER = os.path.expanduser('~/.mozilla/firefox')
+            profile = get_firefox_profile_name(DEFAULT_FF_FOLDER)
+            if profile:
+                FF_BM_DB_PATH = '~/.mozilla/firefox/{}/places.sqlite'.format(profile)
+        elif sys.platform.startswith('freebsd'):
```

Comment:
If the defaults are same as that on Linux please add a simple:

    if sys.platform.startswith('linux') or sys.platform.startswith('freebsd'):